### PR TITLE
Issue 105 document release workflow

### LINKED ISSUE #105: Document RELEASE workflow
Add a REALEASE.md file explaining all the steps that need to be done to make a new release.

### PR DESCRIPTION
**Pull request by JDTheRipperPC**
Resolves #105 

Added RELEASE.md documenting the releasing process.

### CODE REVIEW

File: RELEASE.md [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+
+Execute ALL the tests and linting, tests must end with no errors:
+
+```bash
+make test-all
+```
+
+This command will use tox to execute the unittests with different environments, see tox.ini configuration.
+
+To be able to run this you will need the differents python versions used in the tox.ini file.
```
Comment (csala) [inline RELEASE.md]: differents -> different

File: RELEASE.md [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+============================ 169 passed, 1 skipped, 3 warnings in 7.10s ============================
+flake8 copulas tests examples
+isort -c --recursive copulas tests examples
+```
+
+The execution has finished with no errors, 1 test skipped and 3 warnings.
+		
+## Documentation
+
+The documentation must be up to dates and generated with:
```
Comment (csala) [inline RELEASE.md]: dates -> date

File: RELEASE.md [hunk 3]
```diff
@@ -0,0 +1,183 @@
...
+
+```bash
+make view-docs
+```
+
+Read the documentation to ensure all the changes are reflected in the documentation.
+
+Alternatively, you can simple generate the documentation using the command:
```
Comment (csala) [inline RELEASE.md]: simple -> simply

File: RELEASE.md [hunk 4]
```diff
@@ -0,0 +1,183 @@
...
+
+```bash
+make docs
+```
+
+## HISTORY.md
+
+HISTORY.md is updated with the issues of the milestone:
```
Comment (csala) [inline RELEASE.md]: Make sure ...

File: RELEASE.md (hunk 1 above)
Comment (csala) [reply RELEASE.md]: This was the opposite: Say different where it said differents :-)

**Comment by JDTheRipperPC** [comment]
Requested changes also updated in RDT:
https://github.com/HDI-Project/RDT/pull/95

**Pull request merged**